src/app/attendance/actions/update-attendance.ts

```diff
@@ -0,0 +1,26 @@
+"use server";
+
+import { Tables } from "@/lib/db";
+import { getSupabaseClient } from "@/utils/supabase/supabaseClient";
+import { AttendanceFormValues } from "../components/AttendanceFormBuilder";
+import { redirect } from "next/navigation";
+import { Routes } from "@/lib/routes";
+
+interface Attendance extends AttendanceFormValues {
+  id: string | string[];
```

Comment:
Whoa, why would id be an array of string?

Reply:
Fixed

Reply:
Makes no sense. You'll still need to pass a single string here.

Reply:
Fixed